server/src/main/java/dev/slimevr/tracking/processor/skeleton/ViveEmulation.java

```diff
@@ -84,61 +84,65 @@ public void update() {
 				flyingBack = true;
 				overShooting = true;
 				flySpeed = FLY_BACK_SPEED;
-				getOvershootPosition(targetPosition);
+				targetPosition = getOvershootPosition();
 				return;
 			}
-			skeleton.computedWaistTracker.position.set(getFlyingPos());
-			skeleton.computedWaistTracker.rotation.set(getFlyingRotation());
+			skeleton.computedHipTracker.setPosition(getFlyingPos());
+			skeleton.computedHipTracker.setRotation(getFlyingRotation());
 		}
 		// in this state the tracker is returning to its position but has not
 		// yet overshot its target
 		else if (flyingBack && overShooting) {
-			if (targetPosition.distanceSquared(lastPosition) < NEARLY_ZERO * NEARLY_ZERO) {
+			if (targetPosition.minus(lastPosition).lenSq() < NEARLY_ZERO * NEARLY_ZERO) {
```

Comment:
This is mathematically fine, but I want `b - a` order for clarity.
```suggestion
			if (lastPosition.minus(targetPosition).lenSq() < NEARLY_ZERO * NEARLY_ZERO) {
```